scripts/sequences.js

```diff
@@ -295,6 +295,8 @@ chrome.runtime.onMessage.addListener(function(message, sender, sendResponse) {
 						var anc_arr = nodeArray;
 						// Data join; key function combines name and depth (= position in sequence).
 						var trail = document.getElementById("sequence");
+                        
+                        var textElem = document.createElement('span');
```

Comment:
You see how the identation is completely wrong here? Maybe it has something to do with your editor? Please fix it, all modifications in this file are totally off regarding identation.